tests/unit/test_charm.py

```diff
@@ -113,60 +101,56 @@ def test_on_leader_elected(self):
             self.charm._peers.data[self.charm.app].get("postgres-password", None), password
         )
 
-    @patch("charm.PostgresqlCluster.bootstrap_cluster")
+    @patch("charm.Patroni.bootstrap_cluster")
+    @patch("charm.PostgresqlOperatorCharm._replication_password")
     @patch("charm.PostgresqlOperatorCharm._get_postgres_password")
```

Comment:
You can skip lines 115 and 116 by setting `return_value` in the decorator: 

```suggestion
    @patch("charm.PostgresqlOperatorCharm._replication_password", return_value="fake-replication-pw")
    @patch("charm.PostgresqlOperatorCharm._get_postgres_password", return_value="fake-postgres-pw")
```